p3/requirements.txt

```diff
@@ -1,41 +1,52 @@
-arrow==0.17.0
-astroid==2.4.2
-attrs==20.3.0
+arrow==1.1.0
+astroid==2.5.6
+attrs==21.2.0
 beautifulsoup4==4.9.3
 bs4==0.0.1
 certifi==2020.12.5
 chardet==4.0.0
-click==7.1.2
-Flask==1.1.2
-html5validator==0.3.3
+click==8.0.0
+colorama==0.4.4
+configparser==5.0.2
+crayons==0.4.0
+fancycompleter==0.9.1
+Flask==2.0.0
+html5validator==0.4.0
 idna==2.10
 importlib-metadata==1.7.0
 iniconfig==1.1.1
-isort==5.7.0
-itsdangerous==1.1.0
-Jinja2==2.11.3
-lazy-object-proxy==1.4.3
-MarkupSafe==1.1.1
+isort==5.8.0
+itsdangerous==2.0.0
+Jinja2==3.0.0
+lazy-object-proxy==1.6.0
+MarkupSafe==2.0.0
 mccabe==0.6.1
 more-itertools==8.5.0
 packaging==20.9
+pdbpp==0.10.2
+pkg-resources==0.0.0
```

Comment:
This seems to be breaking P3 install step https://github.com/eecs485staff/p3-insta485-clientside/runs/2604750398.

Reply:
Also, I don't expect pdbpp to be part of the production dependencies.  Did you create a new clean virtual environment before dumping the new deps?

Reply:
I had forgotten to do that the first time around, but I redid it just now and it is working locally. I'll push my changes here after getting the P3 repo dependencies pushed

Reply:
Yeah I usually create a clean virtual env before dumping them.